### ISSUE TITLE
Water balance in/out is switched

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-results-analysis#444]
See image: in/out in this graph should be switched

![image (12)](https://user-images.githubusercontent.com/1453159/67963272-a0d3a280-fbfe-11e9-81e6-5e31654ff345.png)

**Comment by leendertvanwolfswinkel** [comment]
! Is still not fixed

**Comment by Patrick-Nijman** [comment]
@leendertvanwolfswinkel please provide more information. What needs to be switched. How can we reproduce? How can Lukasz generate this graph?

**Comment by leendertvanwolfswinkel** [comment]
I tried to test this, but now I cannot even get a water balance at all. 

Data + screen capture can be downloaded here: https://we.tl/t-goPDod6JTr

I get the error below:

```
Er is een fout opgetreden bij het uitvoeren van Python code: 

numpy.ma.core.MaskError: Mask and data not compatible: data size is 0, mask size is 69. 
Traceback (most recent call last):
  File "C:\Users/leendert.vanwolfswin/AppData/Roaming/QGIS/QGIS3\profiles\test 3Di Toolbox 20220523/python/plugins\ThreeDiToolbox\tool_water_balance\views\waterbalance_widget.py", line 1152, in toggle_polygon_button
    self.update_wb()
  File "C:\Users/leendert.vanwolfswin/AppData/Roaming/QGIS/QGIS3\profiles\test 3Di Toolbox 20220523/python/plugins\ThreeDiToolbox\tool_water_balance\views\waterbalance_widget.py", line 1170, in update_wb
    ts, graph_series = self.calc_wb_graph(
  File "C:\Users/leendert.vanwolfswin/AppData/Roaming/QGIS/QGIS3\profiles\test 3Di Toolbox 20220523/python/plugins\ThreeDiToolbox\tool_water_balance\views\waterbalance_widget.py", line 1230, in calc_wb_graph
    ts, total_time = self.calc.get_aggregated_flows(
  File "C:\Users/leendert.vanwolfswin/AppData/Roaming/QGIS/QGIS3\profiles\test 3Di Toolbox 20220523/python/plugins\ThreeDiToolbox\tool_water_balance\tools\waterbalance.py", line 840, in get_aggregated_flows
    td_vol = ma.masked_array(vol_current, mask=mask_2d_nodes).sum()
  File "C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\numpy\ma\core.py", line 2904, in __new__
    raise MaskError(msg % (nd, nm))
numpy.ma.core.MaskError: Mask and data not compatible: data size is 0, mask size is 69.


Python versie: 3.9.5 (tags/v3.9.5:0a7dcbd, May  3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] 
QGIS versie: 3.22.3-Białowieża Białowieża, 1628765ec7 

Python pad:
C:/PROGRA~1/QGIS32~1.3/apps/qgis/./python
C:/Users/leendert.vanwolfswin/AppData/Roaming/QGIS/QGIS3\profiles\test 3Di Toolbox 20220523/python
C:/Users/leendert.vanwolfswin/AppData/Roaming/QGIS/QGIS3\profiles\test 3Di Toolbox 20220523/python/plugins
C:/PROGRA~1/QGIS32~1.3/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.22.3\bin\python39.zip
C:\PROGRA~1\QGIS32~1.3\apps\Python39\DLLs
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib
C:\Program Files\QGIS 3.22.3\bin
C:\PROGRA~1\QGIS32~1.3\apps\Python39
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\GDAL-3.4.1-py3.9-win-amd64.egg
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\win32
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\win32\lib
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\Pythonwin
```

**Comment by leendertvanwolfswinkel** [comment]
In/out thing is fixed; moved the other problem I ran into to #634
